src/open-hackathon-server/Kaiyuanshe.OpenHackathon.Server/Controllers/UserController.cs

```diff
@@ -65,5 +67,24 @@ public class UserController : HackathonControllerBase
             return Ok(userInfo);
         }
         #endregion
+
+        #region GetUploadToken
+        /// <summary>
+        /// Get file upload token
+        /// </summary>
+        /// <returns>The team</returns>
+        /// <response code="200">Success. The response return a token.</response>
+        [HttpPut]
+        [ProducesResponseType(typeof(Team), StatusCodes.Status200OK)]

```

Comment:
The response type is not `typeof(Team)`. This line is auto-generated swagger, please make is right